mcp_hydrolix/logging_utils.py

```diff
@@ -0,0 +1,49 @@
+"""Logging utilities for redacting sensitive information from logs."""
+
+import logging
+import re
+
+from .auth import TOKEN_PARAM
+
+
+class AccessLogTokenRedactingFilter(logging.Filter):
+    """
+    Filter that redacts token query parameters from uvicorn access logs.
+
+    This filter is specifically designed to intercept log messages that contain
+    request URLs with query parameters and replace token values with [REDACTED].
```

Comment:
should the `Bearer` token from the `Authorization` header also be redacted?

Reply:
The `Authorization` header won't be logged by default, unlike the query parameters.